src/msg/async/AsyncConnection.cc

```diff
@@ -2064,15 +2064,7 @@ void AsyncConnection::fault()
   replacing = false;
   is_reset_from_peer = false;
   outcoming_bl.clear();
```

Comment:
if just remove the stop path, the connection state will not changed means will stay in current state,  and in asyncConneciton::process, the prev_state will equal state, the while loop will break, no chance to go to next state anymore, so does remove the stop path solve the issue?

Reply:
> no effect, lossy connection already exit, see the front of this function

but the lossy connection is not a lossy connection before accepting ready. and the connection can not reconnect most time because of peer address lose or peer type lose.